front-end/cypress/e2e/pages/loginPage.ts

```diff
@@ -73,6 +73,7 @@ function loginDotGovLogin() {
   cy.contains('Manage reports').should('exist');
 
   // Create second create admin after logging in to make pop-up go away
+  cy.wait(500);
```

Comment:
We don't want to use .wait() commands because these often fail when run in CircleCI.  I'd recommend using .should('not.exist') on some element that is exclusive to the  create-a-second-admin modal.  There's an example of this on line 87

Reply:
This would not fix it, but I've got an idea involving waiting for the request to /committee-members/ to resolve.  I'm going to take a quick stab at it

Reply:
Resolved!